mozdef_util/mozdef_util/elasticsearch_client.py

```diff
@@ -85,6 +85,9 @@ def create_alias_multiple_indices(self, alias_name, indices):
     def get_alias(self, alias_name):
         return self.es_connection.indices.get_alias(index='*', name=alias_name).keys()
 
+    def get_aliases(self):
```

Comment:
Since we're modifying mozdef_util, we need to:
1. Bump the mozdef-util version here https://github.com/mozilla/MozDef/blob/master/mozdef_util/setup.py#L59 
2. Publish the new mozdef-util library to pypi (there are make targets for this in https://github.com/mozilla/MozDef/blob/master/mozdef_util/Makefile)
3. Update https://github.com/mozilla/MozDef/blob/master/requirements.txt#L34 to point to the new mozdef_util version

Reply:
Updated in https://github.com/mozilla/MozDef/pull/1186/commits/c4f553d207839204bc46c191a1564fd9ae3ab059 and https://github.com/mozilla/MozDef/pull/1186/commits/35e9c0d4dc00b474f904dde0f133c32ccb6157bc